src/test/java/com/eternalcode/discordapp/review/GitHubReviewUtilTest.java

```diff
@@ -71,13 +72,26 @@ private void assertNotPullRequestUrl(String url) {
         assertTrue(result.isErr());
     }
 
-    @Test
+    @TestFactory
     @DisplayName("Test isPullRequestTitleValid with invalid and valid titles")
-    void testIsPullRequestTitleValid() {
-        assertTrue(GitHubReviewUtil.isPullRequestTitleValid("GH-123 This is a valid title"));
-        assertFalse(GitHubReviewUtil.isPullRequestTitleValid("This is an invalid title"));
-        assertFalse(GitHubReviewUtil.isPullRequestTitleValid("GH- This title has an invalid number"));
-        assertTrue(GitHubReviewUtil.isPullRequestTitleValid("GH-123 Another valid title"));
+    Collection<DynamicTest> testIsPullRequestTitleValid() {
```

Comment:
takie dla valid

return Stream.of("GH-123 This is a valid title", "GH-123 Another valid title")
.map(title -> dynamicTest("Valid Title: " + title, () -> assertTrue(GitHubReviewUtil.isPullRequestTitleValid(title))))

podobnie dla invalid